app/components/channel_list/__snapshots__/channel_list.test.tsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/channel_list should match snapshot 1`] = `
+<RNCSafeAreaProvider
```

Comment:
This does not seems to be testing the actual component, but the safe area provider.

Reply:
@shazm You can provide `initialSafeAreaInsets={{ top: 1, left: 2, right: 3, bottom: 4 }}` to `SafeAreaProvider` in your test to render nested components, e.g.

```
    it('should match snapshot', () => {
        const wrapper = renderWithEverything(
            <SafeAreaProvider initialSafeAreaInsets={{ top: 1, left: 2, right: 3, bottom: 4 }}>
```